db.js

```diff
@@ -0,0 +1,36 @@
+const { MongoClient } = require("mongodb");
+
+module.exports = async function () {
```

Comment:
Would be good to give this function a name

```suggestion
module.exports = async function connect() {
```